marimbabot_vision/launch/marimbabot.launch

```diff
@@ -3,5 +3,13 @@
      <rosparam>
        cv_camera/device_id: 0
      </rosparam>
+      <rosparam>
+       cv_camera/rate: 1
```

Comment:
Idk but we could run the cam at a higher rate, so the delay is reduced a bit.

Reply:
1 was just a safe bet. I can adjust that

Reply:
Okay

Reply:
I would go for 5 in the beginning